app/assets/javascripts/reports/new.js

```diff
@@ -1184,6 +1184,15 @@ function reportHandsonTableConverter() {
         let template = $('#templateSelector').val();
         $.get($('#templateSelector').data('valuesEditorPath'), { template: template }, function(result) {
           $('.report-template-values-container').html(result.html);
+
+          $.each($('.report-template-values-container .values-container select'), function(i, select) {
```

Comment:
I think it is already initialized

Reply:
It is for dropdowns inside template values (user selection)

Reply:
Ah i see. I will rebase.